<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorBarDialog</class>
 <widget class="QDialog" name="ColorBarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ColorBarDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelParameterName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>parameter Name: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelParameterRange">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>parameter Range:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="textParameterRange">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lableColorBarmin">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>colorBarRangeMin:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="textColorBarmin">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelColorBarmax">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>colorBarRangeMax:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="textColorBarMax">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_Origin_y">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>color Number: </string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="textColorNumber">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="nameComboBox">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorBarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorBarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
